package.json

```diff
@@ -45,32 +45,42 @@
     "gatsby-remark-images": "^3.3.33"
   },
   "dependencies": {
+    "@carbon/charts": "0.34.11",
+    "@carbon/charts-react": "0.34.11",
+    "@carbon/elements": "^10.22.0",
+    "@carbon/icons": "^10.20.0",
+    "@carbon/icons-react": "^10.20.0",
+    "@carbon/pictograms": "^10.18.0",
+    "@loadable/component": "^5.12.0",
     "@slack/web-api": "^5.11.0",
+    "carbon-components": "^10.23.0",
     "carbon-components-react": "^7.23.0",
+    "change-case": "^4.1.1",
     "classnames": "^2.2.6",
+    "codesandbox": "^2.1.10",
+    "copy-to-clipboard": "^3.2.1",
+    "gatsby": "^2.19.8",
+    "gatsby-image": "^2.4.15",
+    "gatsby-theme-carbon": "^1.27.0",
+    "lodash-es": "^4.17.15",
+    "markdown-it": "^9.0.1",
+    "nanoid": "^2.1.11",
+    "prettier-config-carbon": "^0.4.0",
+    "prop-types": "^15.7.2",
     "react": "^16.8.6",
-    "react-copy-to-clipboard": "^5.0.1"
+    "react-copy-to-clipboard": "^5.0.1",
+    "react-live": "^2.2.1"
```

Comment:
for reviewers, eslint errors said these should be in `dependencies` not `devDependencies`:
<img width="878" alt="Screen Shot 2020-10-29 at 3 24 52 PM" src="https://user-images.githubusercontent.com/32556167/97635655-b3e1a400-1a05-11eb-8117-b0bc23f76db6.png">